src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -520,6 +524,23 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
                 <WorkflowSettings
                   user={user}
                   workflowDag={workflow.selectedDag}
+                  onSettingsSave={() => {
+                    setShowUpdateMessage(true);
+                    // Show toast message for a few seconds and then update the current tab.
+                    setTimeout(() => {
+                      getDagResultDetails(true);
```

Comment:
If we trigger this right away, the toast doesn't have time to show so I added a small 3 second timeout here.